radio/src/targets/common/arm/stm32/sticks_pwm_driver.cpp

```diff
@@ -117,10 +117,26 @@ extern "C" void PWM_IRQHandler(void)
   }
 }
 
+#if defined(CHANNEL_1023)
+
+#define _CHANNEL0     1
+#define _CHANNEL1     0
+#define _CHANNEL2     2
+#define _CHANNEL3     3
+
+#else
+
+#define _CHANNEL0     0
+#define _CHANNEL1     1
+#define _CHANNEL2     3
+#define _CHANNEL3     2
+
+#endif
```

Comment:
I'd need to check, but there is probably a better way to achieve channel mapping (especially with a more meanful `define` name: `CHANNEL_1023` is a bit cryptic, isn't it?).